fxa-oauth-server/lib/grant.js

```diff
@@ -119,6 +119,7 @@ module.exports.generateTokens = async function generateTokens(grant) {
   const access = await db.generateAccessToken(grant);
   const result = {
     access_token: access.token.toString('hex'),
+    user: access.userId.toString('hex'),
```

Comment:
What if we call this [`sub` like we do for an id token](https://github.com/mozilla/fxa-auth-server/blob/9564168b28bcbb441f06dbe00c950e52eed7eea8/fxa-oauth-server/lib/grant.js#L147). At least it'd be consistent. But I see [verify returns user](https://github.com/mozilla/fxa-auth-server/blob/9564168b28bcbb441f06dbe00c950e52eed7eea8/fxa-oauth-server/lib/routes/verify.js#L32). mmm...

Reply:
another idea is to use the `fxa-uid` field